src/selector.js

```diff
@@ -173,7 +173,11 @@ function find( selector, context, results, seed ) {
 	// Try to shortcut find operations (as opposed to filters) in HTML documents
 	if ( !seed ) {
 
-		if ( ( context ? context.ownerDocument || context : preferredDoc ) !== document ) {
+		// Support: IE 11+, Edge 17 - 18+
+		// IE/Edge sometimes throw a "Permission denied" error when strict-comparing
+		// two documents; shallow comparisons work.
+		// eslint-disable-next-line eqeqeq
+		if ( ( context ? context.ownerDocument || context : preferredDoc ) != document ) {
```

Comment:
This line is the only place where I've seen IE/Edge test failures. I though, though, that it may be worth changing all document comparisons to make sure we're not missing some potential errors.

I could also create an internal utility function to compare documents & put the comment there. That would most likely increase the size, though (currently this PR is at -5 bytes).

Reply:
A version with an utility function would be +15 bytes instead of -5: https://github.com/mgol/jquery/commit/53d084000bdb1e68bf78312781a72e2e0db4379c.